blog/themes/3mdeb/layouts/authors/list.html

```diff
@@ -19,23 +19,23 @@
                       <h2>Social media <strong>profiles</strong></h2>
                     {{ end }}
                     {{ with .Params.facebook }}
-                      <p>
-                        <a href="{{ . }}">Follow {{ $.Params.name }} on Facebook</a>
-                      </p>
+                      <div>
+                        <a href="{{ . }}"><img class="social-icon" src="{{ .Site.BaseURL }}/img/facebook.png" height="32" width="32"> Follow {{ $.Params.name }} on Facebook</a>
+                      </div>
                     {{ end }}
                     {{ with .Params.twitter }}
-                      <p>
-                        <a href="{{ . }}">Follow {{ $.Params.name }} on Twitter</a>
-                      </p>
+                      <div>
+                        <a href="{{ . }}"><img class="social-icon" src="{{ .Site.BaseURL }}/img/twitter.png" height="32" width="32"> Follow {{ $.Params.name }} on Twitter</a>
+                      </div>
                     {{ end }}
                     {{ with .Params.linkedin }}
-                      <p>
-                        <a href="{{ . }}">Find {{ $.Params.name }} on LinkedIn</a>
-                      </p>
+                      <div>
+                        <a href="{{ . }}"><img class="social-icon" src="{{ .Site.BaseURL }}/img/linkedin.png" height="32" width="32"> Find {{ $.Params.name }} on LinkedIn</a>
```

Comment:
@artur-rs I coudn't build a site using {{ .Site.BaseURL }} instead of hard link but i will it will works on server site

Reply:
starting the path with `/` in Hugo is equal to `{{ BaseURL }}`, already committed changes